### PR TITLE
Grouping ccip integration test

### PR DESCRIPTION
**Pull request by AnieeG**
## Solution

### CODE REVIEW

**Review by tateexon — APPROVED** [review]

File: integration-tests/ccip-tests/chaos/ccip_test.go [hunk 1]
```diff
@@ -10,8 +10,8 @@ import (
 	"github.com/smartcontractkit/chainlink-testing-framework/utils"
 	"github.com/stretchr/testify/require"
 
-	"github.com/smartcontractkit/chainlink/integration-tests/actions"
-	"github.com/smartcontractkit/chainlink/integration-tests/testsetups"
+	"github.com/smartcontractkit/integration-tests/ccip-tests/actions"
```
Comment (jasonmci) [inline integration-tests/ccip-tests/chaos/ccip_test.go]: Is this url valid? looks like its pointing at a repo named `integration-tests`

Comment (AnieeG) [reply integration-tests/ccip-tests/chaos/ccip_test.go]: good catch. I updated the module name. Reverting it back.
https://github.com/smartcontractkit/ccip/pull/102/files#diff-7c016cae50e2281d7a89e537d58b33cb9e01501820ff9301365fc006c94b712bR1

**Review by matYang — APPROVED** [review]

File: integration-tests/Makefile [hunk 2]
```diff
@@ -202,50 +202,6 @@ test_benchmark_automation: ## Run the automation benchmark tests
...
 	go test -timeout 300m -v -run ^TestAutomationReorg$$ ./reorg -count=1 | tee automation_reorg_run_`date +"%Y%m%d-%H%M%S"`.log
 
-.PHONY: test_smoke_ccip_simulated
-test_smoke_ccip_simulated: ## Run ccip smoke tests with verbose logging
-	DATABASE_URL="postgresql://postgres:node@localhost:5432/chainlink_test?sslmode=disable" \
-	SELECTED_NETWORKS="SIMULATED,SIMULATED_1,SIMULATED_2" \
-	go test -timeout 24h -count=1 -v $(args) -v -run ^TestSmokeCCIPForBidirectionalLane ./smoke
-
-.PHONY: test_load_ccip_simulated
-test_load_ccip_simulated: ## Run ccip load tests with verbose logging
```
Comment (connorwstein) [inline integration-tests/Makefile:212]: do we need to port these as well to ccip-tests/Makefile?

**Review by connorwstein — APPROVED** [review]

File: integration-tests/Makefile (hunk 2 above)
Comment (AnieeG) [reply integration-tests/Makefile:212]: I have plans to update Makefile and ReadMe as part of next PR

**Pull request merged**